### PR TITLE
Increase default RMP ack timeout to spec-defined 200ms

### PR DESCRIPTION
**Pull request by holbrookt**
#### Problem
`CHIP_CONFIG_RMP_DEFAULT_ACK_TIMEOUT_TICK` is currently set to 1ms, but spec says the default should be 200ms.

#### Change overview
Change `CHIP_CONFIG_RMP_DEFAULT_ACK_TIMEOUT_TICK` to 200ms

#### Testing
* This change does not add any new behavior or features, so we can rely on existing tests for verification

### CODE REVIEW

File: src/messaging/ReliableMessageProtocolConfig.h [hunk 1]
```diff
@@ -79,7 +79,7 @@ namespace Messaging {
  *
  */
 #ifndef CHIP_CONFIG_RMP_DEFAULT_ACK_TIMEOUT_TICK
-#define CHIP_CONFIG_RMP_DEFAULT_ACK_TIMEOUT_TICK (1)
+#define CHIP_CONFIG_RMP_DEFAULT_ACK_TIMEOUT_TICK (200)
```
Comment (bzbarsky-apple) [inline src/messaging/ReliableMessageProtocolConfig.h:82]: This value is in ticks, not milliseconds.  A tick is 2^{CHIP_CONFIG_RMP_TIMER_DEFAULT_PERIOD_SHIFT} milliseconds, so what "1" means depends on what that config setting is.

I think it's set to 6 for all our platforms, so a tick is 64 ms.

Comment (turon) [reply src/messaging/ReliableMessageProtocolConfig.h:82]: I think we'll want to to transition to using platform defined `ms_to_ticks` portability method over time.

Comment (holbrookt) [reply src/messaging/ReliableMessageProtocolConfig.h:82]: Why is this value in units of "ticks" anyways? Seems to me that it should be in milliseconds and then converted to ticks as Martin said, however there is no `ms_to_ticks` function currently

Comment (holbrookt) [reply src/messaging/ReliableMessageProtocolConfig.h:82]: @bzbarsky-apple How do you know that's the definition of a tick?

Comment (bzbarsky-apple) [reply src/messaging/ReliableMessageProtocolConfig.h:82]: By code inspection, sadly, in ReliableMessageManager.cpp, of where `CHIP_CONFIG_RMP_DEFAULT_ACK_TIMEOUT_TICK ` is used.

**Pull request closed without merging**